site/docs/components/form-field/index.mdx

```diff
@@ -0,0 +1,12 @@
+---
+title: Form Field
+data:
+  description: Form Field is a wrapper for UI controls that are typically found in a form (for example, Input, Combobox or Radio Button). It is required to make these controls accessible by providing them with a visible label, validation control and states and descriptive text elements.
+  sourceCodeUrl: "https://github.com/jpmorganchase/salt-ds/blob/main/packages/core/src/form-field/FormField.tsx"
+  package:
+    name: "@salt-ds/core"
+    initialVersion: "1.8.0-rc.0"
+  alsoKnownAs: ["Form Input", "Input Control", "Form Element"]
```

Comment:
Given we mentioned "Form Input" and "Input Control", should some of the form controls be added as related components?

Reply:
not sure whether I should put as 'contains' considering they don't **have** to contain them @mikearildbrown ?

Reply:
I think @origami-z  was referring to `alsoKnownAs`, not `contains`.  However, my personal opinion is that I don't think it would be referred to as "Form Radio Button" or "Form Checkbox". I believe most of the controls will be generically called "Inputs". So I think this is fine (if I understood correctly).

Reply:
I didn't realise we don't have true "related" relationship apart from  `similarTo` and `contains`, which I first assumed would be useful to be shown there. Don't worry about this, i will add to doc week feedback, as "related" appears to be useful in many components.

Reply:
Actually already no.23 in the [comment](https://github.com/jpmorganchase/salt-ds/issues/2265#issuecomment-1687856527)